lib/tasks/graphiti.rake

```diff
@@ -0,0 +1,53 @@
+# Undefine tasks from `graphiti` gem to avoid duplicated work.
```

Comment:
Yeah, they're removed in https://github.com/graphiti-api/graphiti/pull/141. I just put this here because defining a task twice will just append so this is one of the few places where I think duplication could actually do something bad.